content/github/authenticating-to-github/telling-git-about-your-signing-key.md

```diff
@@ -30,12 +30,12 @@ If you have multiple GPG keys, you need to tell Git which one to use.
 {% data reusables.gpg.paste-gpg-key-id %}
 1. If you aren't using the GPG suite, paste the text below to add the GPG key to your bash profile:
   ```shell
-  $ test -r ~/.bash_profile && echo 'export GPG_TTY=$(tty)' >> ~/.bash_profile
+  $ test -r ~/.zshrc && echo 'export GPG_TTY=$(tty)' >> ~/.zshrc
   $ echo 'export GPG_TTY=$(tty)' >> ~/.profile
```

Comment:
```suggestion
  test -r ~/.zshrc && echo 'export GPG_TTY=$(tty)' >> ~/.zshrc
  echo 'export GPG_TTY=$(tty)' >> ~/.profile
```

According to [the style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#code-blocks), you should only include the `$` when you are showing the command's output in the same block. 

> Disclaimer: **I am not a GitHub employee.**

Reply:
I left it because it was already in place, but I'm happy to remove them if necessary